content/dota_addons/legion_td/panorama/layout/custom_game/custom_ui_manifest.xml

```diff
@@ -43,12 +43,12 @@
 
 
     var newUI = $.GetContextPanel().GetParent().GetParent().FindChildTraverse("HUDElements").FindChildTraverse("lower_hud").FindChildTraverse("center_with_stats").FindChildTraverse("center_block");
-    var NoGlyphAndRadar = $.GetContextPanel().GetParent().GetParent().FindChildTraverse("HUDElements").FindChildTraverse("minimap_container");
-    var NoTPScrollUI = $.GetContextPanel().GetParent().GetParent().FindChildTraverse("HUDElements").FindChildTraverse("lower_hud").FindChildTraverse("center_with_stats").FindChildTraverse("center_block");
-    var NoKDA = $.GetContextPanel().GetParent().GetParent().FindChildTraverse("HUDElements").FindChildTraverse("stackable_side_panels");
+	var NoGlyphAndRadar = $.GetContextPanel().GetParent().GetParent().FindChildTraverse("HUDElements").FindChildTraverse("minimap_container");
+	var NoTPScrollUI = $.GetContextPanel().GetParent().GetParent().FindChildTraverse("HUDElements").FindChildTraverse("lower_hud").FindChildTraverse("center_with_stats").FindChildTraverse("center_block");
+	var NoKDA = $.GetContextPanel().GetParent().GetParent().FindChildTraverse("HUDElements").FindChildTraverse("stackable_side_panels");
     
 	newUI.FindChildTraverse("StatBranch").style.visibility = "collapse";
-	//NoGlyphAndRadar.FindChildTraverse("GlyphScanContainer").style.visibility = "collapse";
+	NoGlyphAndRadar.FindChildTraverse("GlyphScanContainer").style.visibility = "collapse";
 	NoTPScrollUI.FindChildTraverse("inventory_tpscroll_container").style.visibility = "collapse";
 	NoKDA.FindChildTraverse("quickstats").style.visibility = "collapse";

```

Comment:
Please fix the indention of the whole file, some lines use tabs others use spaces. All should use 4 spaces